From 08dc177c290977417b2aea05d84b75807466c039 Mon Sep 17 00:00:00 2001
From: Huang Rui <ray.huang@amd.com>
Date: Wed, 20 Feb 2019 20:17:23 +0800
Subject: [PATCH 2269/2940] drm/amd/powerplay: modify the feature mask to
 enable gfx/soc dpm

So far, the gfx/soc dpm is enabled with feature mask set.

Signed-off-by: Huang Rui <ray.huang@amd.com>
Reviewed-by: Hawking Zhang <Hawking.Zhang@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/powerplay/navi10_ppt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/powerplay/navi10_ppt.c b/drivers/gpu/drm/amd/powerplay/navi10_ppt.c
index a97072cc0396..3327af2376d7 100644
--- a/drivers/gpu/drm/amd/powerplay/navi10_ppt.c
+++ b/drivers/gpu/drm/amd/powerplay/navi10_ppt.c
@@ -119,8 +119,8 @@ navi10_get_unallowed_feature_mask(struct smu_context *smu,
 	if (num > 2)
 		return -EINVAL;
 
-	feature_mask[0] = 0x0C677844;
-	feature_mask[1] = 0xFFFFFF28; /* bit32~bit63 is Unsupported */
+	feature_mask[0] = 0xffffffe4;
+	feature_mask[1] = 0xffffffff;	/* bit32~bit63 is Unsupported */
 
 	return 0;
 }
-- 
2.17.1

